Gemfile

```diff
@@ -62,11 +62,10 @@ gem "split_accounts", "~> 0.0.1", path: "vendor/engines/split_accounts"
 gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
 
 group :development do
-  gem "better_errors"
-  gem "binding_of_caller"
```

Comment:
If pry works, then, 👍

On Thursday, June 9, 2016, Daniel Rench <notifications@github.com> wrote:

> In Gemfile
> <https://github.com/tablexi/nucore-open/pull/661#discussion_r66552533>:
>
> > @@ -62,11 +62,10 @@ gem "split_accounts", "~> 0.0.1", path: "vendor/engines/split_accounts"
> >  gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
> >
> >  group :development do
> > -  gem "better_errors"
> > -  gem "binding_of_caller"
>
> I used starter's Gemfile as a guideline, and it doesn't include
> binding_of_caller. It looks like only a couple TXI projects include both
> web-console and binding_of_caller in their Gemfile.
>
> pry works for me without it, though I don't know if I'm hitting all its
> features.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tablexi/nucore-open/pull/661/files/579bcb1984bddb757377b0af0cbd461b782c2f40#r66552533>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABDFZy98b24U8ayaV9G615HRYhysE3Rwks5qKMb8gaJpZM4IyfVu>
> .
>